src/test/java/teammates/test/cases/util/TimeHelperTest.java

```diff
@@ -57,44 +57,29 @@ public void testCombineDateTime() {
 
     @Test
     public void testEndOfYearDates() {
-        Calendar cal = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
-        cal.clear();
-        cal.set(2015, 11, 30, 12, 0, 0);
-        Date date = cal.getTime();
+        LocalDateTime date = LocalDateTime.of(2015, Month.DECEMBER, 30, 12, 0);
         assertEquals("30/12/2015", TimeHelper.formatDate(date));
         assertEquals("Wed, 30 Dec, 2015", TimeHelper.formatDateForSessionsForm(date));
         assertEquals("Wed, 30 Dec 2015, 12:00 NOON", TimeHelper.formatTime12H(date));
-        assertEquals("Wed, 30 Dec 2015, 12:00 NOON UTC", TimeHelper.formatDateTimeForSessions(date, 0));
+        assertEquals("Wed, 30 Dec 2015, 12:00 NOON UTC", TimeHelper.formatDateTimeForSessions(
+                date.atZone(ZoneId.of("UTC")).toInstant(), ZoneId.of("UTC")));
```

Comment:
But there's also a hardcoded "UTC" in the expected string, so if we keep using the `Const.DEFAULT_TIME_ZONE` then that part should change to `Const.DEFAULT_TIME_ZONE.getId()`? Anyway, I think it's good to be as specific as possible and reduces dependencies in tests.

Reply:
The expected string here is `"Wed, 30 Dec 2015, 12:00 NOON UTC"` though, so replacing `UTC` with arbitrary time zones will cause the test to fail unless you also changed the expected text.
If we replaced `UTC` with `UTC+01:00`, the formatter being tested here will produce `"Wed, 30 Dec 2015, 12:00 NOON UTC+01:00"`. This wouldn't cause the desired interpretation of the test as you described.

`Const#DEFAULT_TIME_ZONE` also semantically speaking has no preference for a particular zone (it could be `Asia/Singapore` or whatever), so it doesn't make sense to use `Const#DEFAULT_TIME_ZONE` in this test because it cannot be guaranteed to always be UTC.

Am I understanding your comment correctly? I am a bit puzzled.